extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/indexing/error/InsertCannotBeEmptyFault.java

```diff
@@ -38,7 +38,8 @@ public InsertCannotBeEmptyFault(
       @JsonProperty("dataSource") final String dataSource
   )
   {
-    super(CODE, "No rows to insert for dataSource [%s]", dataSource);
+    super(CODE, "No rows to insert for dataSource[%s]. Set failOnEmptyInsert=false"
```

Comment:
typically we use colons like `failOnEmptyInsert: false` since it's more JSON-y, and the stuff typically gets specified in JSON